jim-wang-intel/vision-self-checkout
# feat: add profile gst_capi face_detection

## PR Checklist



## What are you changing?
add feature: gst capi face_detection profile

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
1. make build-gst-capi
2. ./camera-simulator/camera-simulator.sh
3. PIPELINE_PROFILE="face_detection" RENDER_MODE=1 sudo -E ./run.sh --workload ovms --inputsrc rtsp://127.0.0.1:8554/camera_0 --platform core

after above 3 commands, you should see video streaming result of inferencing popped up.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
